docs/developer_guide/pull-request.md

```diff
@@ -0,0 +1,36 @@
+# Creating a Pull Request
+
+Once you've made your changes to the MedModels codebase and are ready to share them, you'll submit a pull request (PR). Here's a guide to walk you through the process:
+
+**1. Push your Local Branch:**
+
+- Commit your changes locally using clear and concise commit messages following the Conventional Commits standard ([https://www.conventionalcommits.org/en/v1.0.0/](https://www.conventionalcommits.org/en/v1.0.0/)).
+- This standard encourages using a specific format for commit messages that categorizes your changes. Here's the breakdown:
+
+  - **Type:** Start your message with type like `feat`, `fix`, `refactor`, `docs`, etc. We use the following types in this project:
+    - `build`: Changes related to build infrastructure or tools.
+    - `chore`: Changes that don't directly affect functionality, like updating dependencies or formatting.
+    - `ci`: Changes related to the continuous integration pipeline.
+    - `docs`: Changes to documentation or comments.
+    - `feat`: Introduction of a new feature or functionality.
+    - `fix`: Bug fixes or resolving issues.
+    - `test`: Adding or improving tests.
+    - `refactor`: Code improvements that don't introduce new features or fix bugs (e.g., improving code structure or readability).
+
+- Push your local branch to your remote repository on GitHub.
+
+**2. Create a Pull Request:**
+
+- Navigate to the MedModels repository on GitHub and go to the "Pull requests" tab.
+- Click on the green "New pull request" button.
+- Select the branch containing your changes from the "head" branch dropdown.
```

Comment:
there is no "head" branch dropdown. it would be the "compare" branch dropdown.

imo we don't need to explain exactly which buttons should be pressed for creating a PR